### PR TITLE
fix: fixing header with a11y and mobile

### PR DESCRIPTION
**Pull request by alejandroxlopez**
Adding some a11y code, and basic mobile support (temporary)

### CODE REVIEW

File: src/components/AppHeader/AppHeader.vue [hunk 1]
```diff
@@ -95,47 +120,128 @@ export default {
...
+    .navbar-burger {
       span {
-        height: 3px;
-        width: 30px;
-        left: calc(50% - 15px);
+        height: 2px;
+      }
+    }
+    h1, h2 {
+      font-family: 'Montserrat', 'Tahoma', sans-serif;
```
Comment (ibdf) [inline src/components/AppHeader/AppHeader.vue]: use bulma variables instead of hard coding it

File: src/components/AppHeader/AppHeader.vue [hunk 2]
```diff
@@ -95,47 +120,128 @@ export default {
...
+      font-weight: 400;
```
Comment (ibdf) [inline src/components/AppHeader/AppHeader.vue]: use bulma variables instead of hard coding it

File: src/components/AppHeader/AppHeader.vue [hunk 3]
```diff
@@ -1,57 +1,82 @@
...
+            <span aria-hidden="true" />
+          </a>
         </div>
-      </div>
-      <div
-        class="level-right"
-        :class="{ 'is-hidden-mobile': showOnBurgerClicked }"
-      >
-        <div class="level-item">
+        <!-- Menu Desktop -->
```
Comment (ibdf) [inline src/components/AppHeader/AppHeader.vue]: Need to remove old comments

**Review by ibdf — CHANGES_REQUESTED** [review]
1. Remove mobile menu from within header
2. make mobile menu it's own components
3. add an extra  named slot mobile-menu to AppHeader

**Review by karissademi — APPROVED** [review]
We discussed this in our biweeky meeting. 👍

**Review by ibdf — APPROVED** [review]

**Pull request merged**